homeassistant/components/media_player/braviatv.py

```diff
@@ -221,7 +221,8 @@ def __init__(self, host, mac, name, pin):
     def update(self):
         """Update TV info."""
         if not self._braviarc.is_connected():
-            self._braviarc.connect(self._pin, CLIENTID_PREFIX, NICKNAME)
+            if self._braviarc.get_power_status() != 'off':
```

Comment:
The connect is a way to authenticate and get a token from the TV (which is stored in a cookie). Maybe there are a few API services that don't need the token to work. I didn't realized that possibility.